bosh_cli/lib/cli/commands/ssh.rb

```diff
@@ -6,7 +6,7 @@ module Command
     class Ssh < Base
 
       # bosh ssh
-      usage 'ssh'
+      usage 'ssh <job>/<index>'
```

Comment:
Currently, `bosh scp` allows the following calling conventions:

    bosh scp box_z1/3 /a /b --upload
    bosh scp box_z1 3 /a /b --upload     # space-separated job and index number

Changing the call signature of `scp(*args)` to either `scp(job, index, source, destination)` or `scp(job_and_index, source, destination)` would seem to force one and drop support for the other.

I suppose we could do something like this:

```
def scp(job, index, source, destination)
  if destination.nil? && job.match('/')
    destination = source
    source = index
    job, index = job.split('/', 2)
  end
  # ...
end
```

I find that a bit more awkward than the current `*args` approach, but if that is preferred to (ab)using the usage string, I don't mind.  I really just want the help screen to reflect accurate usage of `bosh scp`, since I was also initially confused about how to invoke it.